src/graphql/admin/mutations.ts

```diff
@@ -12,23 +12,25 @@ import ColdStorageRebalanceToHotWalletMutation from "@graphql/admin/root/mutatio
 
 import AccountsAddUsdWalletMutation from "./root/mutation/account-add-usd-wallet"
 
-const MutationType = GT.Object({
-  name: "Mutation",
-  fields: () => ({
+export const mutationFields = {
+  unauthed: {
     userRequestAuthCode: UserRequestAuthCodeMutation,
     userLogin: UserLoginMutation,
 
     captchaCreateChallenge: CaptchaCreateChallengeMutation,
     captchaRequestAuthCode: CaptchaRequestAuthCodeMutation,
 
+    accountsAddUsdWallet: AccountsAddUsdWalletMutation,
```

Comment:
should probably be authed

Reply:
I was thinking this as well when I was moving these around. Done https://github.com/GaloyMoney/galoy/pull/2097/commits/5bc7ab6fd86590e6f410a73f4cc3988d26d61c5e